tests/cli-warning-rbac-appid-bug.test.ts

```
@@ -0,0 +1,307 @@
+    w.fatal === true
+    && w.message.includes(RBAC_BUG_MARKER)
+    && w.message.includes(RBAC_BUG_HINT),
+  )
+}
+
+async function cleanupKey(handle: KeyHandle) {
+  await serviceRoleSupabase.from('role_bindings').delete().eq('principal_id', handle.rbacId)
+  await serviceRoleSupabase.from('apikeys').delete().eq('id', handle.id)
+}
+
+describe.skipIf(USE_CLOUDFLARE_WORKERS)('CLI warning: RBAC v2 + limited_to_apps + old CLI (PR #2282)', () => {
```

ESLint

[error] `describe`s should begin with lowercase

(test/prefer-lowercase-title)